### PR TITLE
Change runtime renderer to use a single StringSink for writing

### PR DESCRIPTION
**Pull request by srawlins**

### CODE REVIEW

**Comment by srawlins** [comment]
This failing test looks very unrelated.

**Comment by jcollins-g** [comment]
yep failing test looks like more offset retrieval on synthetic accessors.  I thought I had fixed that, will look into it today.

File: lib/src/mustachio/renderer_base.dart [hunk 1]
```diff
@@ -148,20 +148,21 @@ abstract class RendererBase<T> {
   /// The output buffer into which [context] is rendered, using a template.
   // TODO(srawlins): Pass around a single [StringBuffer], and make this field
```
Comment (jcollins-g) [inline lib/src/mustachio/renderer_base.dart:149]: todo seems obsolete!

**Review by jcollins-g — APPROVED** [review]
very nice!

File: lib/src/mustachio/renderer_base.dart (hunk 1 above)
Comment (srawlins) [reply lib/src/mustachio/renderer_base.dart]: done

**Pull request merged**